README.md

```diff
@@ -16,15 +16,15 @@ Page hits are tracked with Google Analytics, so feel free to use an adblocker. T
 
 ## Contributing
 
-Just create a pull request and we'll get to it. Thanks for helping!
+We use an [efficient branching method](http://nvie.com/posts/a-successful-git-branching-model) when adding features. If you would like to contribute, please [contact us](https://openscopeatc.slack.com/) and we'll help as soon as we can! Thank you!
```

Comment:
this should be the `GitFlow Branching Model`.  and the slack app link is http://slack.openscope.co/